File: app/src/main/java/com/abeimaso/skinsis/SplashScreenActivity.java
package com.abeimaso.skinsis;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

public class SplashScreenActivity  extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getSupportActionBar().hide();
        setContentView(R.layout.activity_splashscreen);
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent(SplashScreenActivity.this, SignInActivity.class);
                startActivity(intent);
                finish();
            }
        }, 3000);

    }
}

File: app/src/main/java/com/abeimaso/skinsis/SplashScreenActivity.kt
package com.abeimaso.skinsis

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.abeimaso.skinsis.R
import android.content.Intent
import android.os.Handler
import com.abeimaso.skinsis.SignInActivity

class SplashScreenActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportActionBar!!.hide()
        setContentView(R.layout.activity_splashscreen)
        Handler().postDelayed({
            val intent = Intent(this@SplashScreenActivity, SignInActivity::class.java)
            startActivity(intent)
            finish()
        }, 3000)
    }
}